src/screens/HomeScreen.tsx

```diff
@@ -47,6 +47,8 @@ export const HomeScreen: React.FC<HomeScreenProps> = ({ navigation }) => {
   const isFocused = useIsFocused()
   const notificationListener = useRef<Subscription>()
   const responseListener = useRef<Subscription>()
+  const lastNotificationResponse = Notifications.useLastNotificationResponse()
+  const [initialLoadingComplete, setInitialLoadingComplete] = useState(false)
```

Comment:
one Q: if we store this state in the component and we navigate away from this screen, the next time we return to this screen the state will be reinitialized to false. Is this what we expect?

Reply:
I agree that conceptually this is wrong. On the other hand there's a chance that, e.g. because of how the app navigation works, this may not be an issue in practice. I would nevertheless look into it to either confirm that it's not an issue or find a different solution.

Reply:
resolved in 519d0d67de4d0b9a131685cc1d39419f3e0efd1e